docs/ConfigurationGuide.md

```diff
@@ -164,3 +164,48 @@ After loading the python file during the service startup each new user
 defined variable is logged for confirmation. Any failures to load
 these changes will be logged. If no new definitions are found
 indication of this will also be logged
+
+# User Customized QR Page
+
+## ConfigMap Based Overrides
+
+By default the QR page shown to users uses OpenWallet branding users
+are capable of overriding any of these files using the
+`controller.htmlFileOverrides` value in
+[values.yaml](charts/vc-authn-oidc/values.yaml).
+These files will override the existing files seen in
+[html-templates](../html-templates/).
```

Comment:
Please expand documentation to explain how to use the `customHtmlConfigMapName` to turn on configmap volume substitutions, and what are the expected keys for the configmap to work correctly (i.e.: the file names that will be replaced).